src/handlers/gestures/GestureDetector.tsx

```diff
@@ -573,12 +574,31 @@ function validateDetectorChildren(ref: any) {
   }
 }
 
+const applyUserSelectProp = (
+  userSelect: UserSelect,
+  gesture: ComposedGesture | GestureType
+): void => {
+  if (gesture instanceof ComposedGesture) {
+    gesture.toGestureArray().forEach((gesture) => {
+      gesture.config.userSelect = userSelect;
+    });
+  } else {
+    gesture.config.userSelect = userSelect;
+  }
```

Comment:
There's no need to have a conditional here. Simple gestures also have `toGestureArray` implemented: https://github.com/software-mansion/react-native-gesture-handler/blob/49709ad072a0fac79bd9324d3f57f8550db05c79/src/handlers/gestures/gesture.ts#L292-L294

Reply:
Oh, i missed that. It is changed now.